content/en/docs/architecture/timeouts.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Timeouts of a Testrun"
+date: 2020-12-21T23:27:39-04:00
+draft: false
+---
+
+We have timeouts for different aspects of a testrun. Some of them are applied to all jobs while the others are configurable in jobs' definition.
```

Comment:
I think we need a broad overview of how this works instead of just a list of items, with language that explains what users can expect at the different stages and _which knobs we expect users to touch_. See https://github.com/kubernetes/test-infra/pull/20093/files for an example.

Reply:
Updated with my understanding.

It is unclear to me if we expect users to set up specific values of `timeout` and `grace_period` for their `org/repo`?